README.md

```
@@ -1,13 +1,33 @@
-# Seminar Website
+<div align="center">
+  <p>
+    <img src="public/seminar-logo.svg" width="50%" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)